ra/ra.go

```diff
@@ -1837,7 +1837,9 @@ func (ra *RegistrationAuthorityImpl) recordValidation(ctx context.Context, authI
 // countFailedValidation increments the failed authorizations per domain per
 // account rate limit. There is no reason to surface errors from this function
 // to the Subscriber, spends against this limit are best effort.
```

Comment:
You're going to need to update this documentation to describe the new things this function is also doing.